server/src/modules/admin/admin.controller.ts

```diff
@@ -163,8 +164,8 @@ export class AdminController {
     @ApiOperation({ description: 'Retrieve a list of reports' })
     @ApiOkPaginatedResponse(ReportDto, { description: 'Paginated list of reports' })
     @ApiBadRequestResponse({ description: 'Invalid query data' })
-    getReports(/* @Query() query: some query, check old API */): Promise<PaginatedResponseDto<ReportDto>> {
-        return void 0;
+    getReports(@Query() query?: PaginationQuery): Promise<PaginatedResponseDto<ReportDto>> {
```

Comment:
Sorry it looks like I missed a query param when stubbing this, old API takes a `resolved` param
```js
// validations/admin.js
	getReports: {
		[Segments.QUERY]: Joi.object().keys({
			limit: validation.queryParam.limit,
			offset: validation.queryParam.offset,
			resolved: validation.report.resolved,
		}),
	}
```
so need a dedicated query and to handle service logic for that